app/controllers/donors_controller.rb

```diff
@@ -9,9 +9,31 @@ def index
     @donors = Donor.all
   end
 
+  def update
+    @donor = find_donor(params[:id])
+
+    if @donor.update_attributes(donor_params)
```

Comment:
There's no point in using the longer `#update_attributes`, because it's just an alias for `#update`. 😀 

if @donor.update(donor_params)